Tests/E2E/CupsTests.cs

```diff
@@ -67,7 +74,8 @@ public async Task Test_Concentrator_Can_Receive_Updates_Then_Connect_To_Lns_And_
                 // Waiting 30s for being sure that BasicStation actually started up
                 await Task.Delay(30_000);
 
-                var expectedLog = stationEui + ": Received 'version' message for station";
+                // If package log does not match, firmware upgrade process failed
+                var expectedLog = stationEui + $": Received 'version' message for station '{TestFixture.Configuration.CupsBasicStationVersion}' with package '{TestFixture.Configuration.CupsBasicStationPackage}'";
```

Comment:
As discussed offline, it would be nice to have an additional check one the reported twin properties to make sure the version was updated there. This can be done in a follow-up PR because we don't update reported version property yet after firmware upgrade.

Reply:
Thanks, I created #1345  for working on this